Source/WebCore/Modules/webcodecs/WebCodecsAudioEncoder.cpp

```diff
@@ -92,6 +102,12 @@ static bool isValidEncoderConfig(const WebCodecsAudioEncoderConfig& config)
     if (StringView(config.codec).trim(isASCIIWhitespace<UChar>).isEmpty())
         return false;
 
+    if (!config.sampleRate || !config.numberOfChannels)
+        return false;
+
+    if (config.bitrate && *config.bitrate > std::numeric_limits<int>::max())
```

Comment:
Do we need this fix or should we instead update the WebIDL to add EnforceRange on the various properties instead?

Reply:
That's what the spec states indeed. However this is existing, I could add this as a follow-up

Reply:
the IDL already has them as [EnforceRange] but this is checking that it's not > int::max() ; while the spec has it has a "long long"  I will lodge a WPT bug.

Reply:
https://github.com/web-platform-tests/wpt/issues/49634